lesson24/register-done.html

```diff
@@ -0,0 +1,2 @@
+<h1>会員登録が完了しました</h1>
+<p>登録いただいたメールアドレス宛に登録内容をお送りしましたのでログインして下さい。</p>
```

Comment:
It's garbled, I think it is because the file of register-done.html does not declare! DOCTYPE HTML.

Reply:
Wow it's a terrible mistake and an embarrassment... 😭 Thank you so much for telling me!!
[e2b40c8](https://github.com/yuka830/js-lessons/pull/29/commits/e2b40c89561a7600626e24eb6256afed13f4b437)